tests/e2e-http-server/http-custom-certs/50-verify-metrics.yaml

```diff
@@ -27,7 +27,7 @@ data:
     set -o errexit
 
     # metrics can change only validate a portion of it
-    curl --insecure --fail --verbose --key /certs/tls.key --cert /certs/tls.crt https://v-http-custom-certs-main-0.v-http-custom-certs:8443/metrics | tee /tmp/curl.out
+    curl --insecure --fail --verbose --key /certs/tls.key --cert /certs/tls.crt https://v-http-custom-certs-main-0.v-http-custom-certs:8443/v1/metrics | tee /tmp/curl.out
```

Comment:
We still need this test to be compatible when run against 12.0.3 image. We can't really rely on the vertica version since 12.0.4-* will have both old and new API. Can we try both endpoints and make sure that one works?